indico/modules/events/contributions/blueprint.py

```diff
@@ -37,3 +40,15 @@
                  'manage_update_contrib', RHEditContribution, methods=('GET', 'POST'))
 _bp.add_url_rule('/manage/contributions/<int:contrib_id>',
                  'manage_contrib_rest', RHContributionREST, methods=('DELETE', 'PATCH'))
+
+# Subcontributions
+_bp.add_url_rule('/manage/contributions/<int:contrib_id>/subcontributions', 'manage_subcontributions',
+                 RHContributionSubContributions)
+_bp.add_url_rule('/manage/contributions/<int:contrib_id>/subcontributions/delete', 'manage_delete_subcontribs',
```

Comment:
why not `DELETE /manage/contributions/<int:contrib_id>/subcontributions/<int:subcontrib_id>`?